types/mongodb/index.d.ts

```diff
@@ -995,19 +995,19 @@ export interface Collection<TSchema = Default> {
     initializeUnorderedBulkOp(options?: CommonOptions): UnorderedBulkOperation;
     /** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#insertOne */
     /** @deprecated Use insertOne, insertMany or bulkWrite */
-    insert(docs: TSchema, callback: MongoCallback<InsertOneWriteOpResult>): void;
+    insert(docs: Omit<TSchema, '_id'> & { _id?: any }, callback: MongoCallback<InsertOneWriteOpResult>): void;
```

Comment:
I think defining a separate type function like the one described in [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33740#issuecomment-508139851) makes the code much more compact.